app/assets/javascripts/live_search.js

```diff
@@ -187,40 +226,41 @@
    * rewrite the appropriate tracking data attribute to delineate the group and document index.
    */
   LiveSearch.prototype.indexTrackingData = function indexTrackingData () {
-    var $groupEls = $('.filtered-results__group')
+    var $groupEls = document.querySelectorAll('.filtered-results__group')
     if ($groupEls.length > 0) {
-      $groupEls.each(function (groupIndex) {
-        var $resultEls = $(this).find('.gem-c-document-list__item')
-        $resultEls.each(function (documentIndex) {
-          var $document = $(this)
-          var $documentLink = $document.find('.gem-c-document-list__item-title')
-          var trackingAction = $documentLink.attr('data-track-action')
+      for (var g = 0; g < $groupEls.length; g++) {
+        var $resultEls = $groupEls[g].querySelectorAll('.gem-c-document-list__item')
+
+        for (var d = 0; d < $resultEls.length; d++) {
+          var $document = $resultEls[d]
+          var $documentLink = $document.querySelector('.gem-c-document-list__item-title')
+          var trackingAction = $documentLink.getAttribute('data-track-action')
           trackingAction = trackingAction.replace(/\.\d+$/, '')
-          trackingAction = [trackingAction, groupIndex + 1, documentIndex + 1].join('.')
-          $documentLink.attr('data-track-action', trackingAction)
-        })
-      })
+          trackingAction = [trackingAction, g + 1, d + 1].join('.')
+          $documentLink.setAttribute('data-track-action', trackingAction)
+        }
+      }
     }
 
-    var $results = $('.js-finder-results')
+    var $results = document.querySelector('.js-finder-results') || []
     if ($results.length > 0) {
```

Comment:
I think the array fallback is unnecessary here, unless I'm missing something.

```suggestion
    var $results = document.querySelector('.js-finder-results')
    if ($results) {
```

Reply:
Yes, with the same logic applied to the `if ($mostRelevantDocumentLink) {` line that follows.